Tests/AppStoreConnectClientTests/AppStoreConnectClientTests.swift

```diff
@@ -37,41 +37,65 @@ final class AppStoreConnectClientTests: XCTestCase {
     }
     
     func testFetchAppsBadRequest() async throws {
+        let statusCode = 400
         var mockClient = MockAPIClient()
         mockClient.result = .badRequest
         let client = AppStoreConnectClient(client: mockClient)
-        
+        let text = """
+            \nThe request failed with: \(statusCode), BAD_REQUEST, Invalid Input, The provided data failed validation.
```

Comment:
We either need to add this format string to some Contacts or avoid comparing errors (instead just catch particular error in separate catch block)

Reply:
Changed.